src/ucp/core/ucp_ep.c

```diff
@@ -71,6 +71,17 @@ typedef struct ucp_ep_discard_lanes_arg {
 } ucp_ep_discard_lanes_arg_t;
 
 
+/**
```

Comment:
one space line in C file

Reply:
but I see that we use two spaces between structures in this file